pygitguardian/config.py

```diff
@@ -5,3 +5,55 @@
 MULTI_DOCUMENT_LIMIT = 20
 DOCUMENT_SIZE_THRESHOLD_BYTES = 1048576  # 1MB
 MAXIMUM_PAYLOAD_SIZE = 2621440  # 25MB
+
+
+DEFAULT_REWRITE_GIT_HISTORY_MESSAGE = """  To prevent having to rewrite git history in the future, setup ggshield as a pre-commit hook:
+     https://docs.gitguardian.com/ggshield-docs/integrations/git-hooks/pre-commit\n"""
+
+DEFAULT_PRE_COMMIT_MESSAGE = """> How to remediate
+
+  Since the secret was detected before the commit was made:
+  1. replace the secret with its reference (e.g. environment variable).
```

Comment:
Should we take advantage of this to reword a tad the `its reference` ? Something like: `1. Fetch the secret's value instead`

In anyway, capitalize the instruction.

Reply:
We could but then I'll also have to modify the default message in GIM, I could also ask Mathieu ?